packages/flutter/lib/src/widgets/_web_image_web.dart

```diff
@@ -366,3 +390,20 @@ class RenderWebImage extends RenderShiftedBox {
     );
   }
 }
+
+extension on BoxFit {
+  web.CSSObjectFit get cssObjectFit {
+    switch (this) {
+      case BoxFit.contain:
```

Comment:
Why are these values all mapped to `cover`? Shouldn't we at least map `contain`?